### DISCUSSION TITLE
Variables

### DISCUSSION THREAD

**Discussion by jordi-z80** [rgthree/rgthree-comfy#330] [Q&A]
Is there something like a programming "variable" in the nodes?

I'm a programmer and I'm having a hard time dealing with the nodes, but I'm very used to variables. In example, I have 8 output images and I want them side by side. Even with helping tools, I still see lots of nodes dancing around.

What I'd love to be able to do is to just assign the image output of the VAE decoder to "image1" (variable writer node), and then, in another zone of the workflow, be able to have a "variable reader" where I could put "image1" in there, so the nodes would be virtually connected, without the nodes but clearly you know where your node comes from.

**Comment by DrJKL** [comment]
You might be able to use something like Use Everywhere or the Set/Get in KJNodes.

**Reply by jordi-z80** [reply]
Thanks, I'll check get/set KJNodes to see it they fit my needs (Use Everywhere unfortunately don't)
